site_root/input/resources/examples/DocumentReference-example-document-reference-id.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Shouldn't this example resource be updated using `ncpi-drs-document-reference`, `ncpi-drs-attchment`, and `drs-uri`?

Reply:
@liberaliscomputing   


[The example](https://github.com/ncpi-fhir/ncpi-model-forge/blob/drs/site_root/input/resources/examples/DocumentReference-example-document-reference-id.json#L6 ) already has `ncpi-drs-document-reference`.   Did I miss something?

Re. `ncpi-drs-attchment, and drs-uri` AFAIK, since an Attachment is a Datatype, not a Resource it does not have the `meta.profile` to tag the extension.  Is there another mechanism?

Reply:
Unclear to me how to specify datatype of inline constraints.   I think we should be good based on the single explicit declaration of `ncpi-drs-document-reference`

@liberaliscomputing